toolchain/check/testdata/class/generic/member_access.carbon

```diff
@@ -191,10 +193,11 @@ fn StaticMemberFunctionCall(T:! type) -> Class(T) {
 // CHECK:STDOUT: !definition:
 // CHECK:STDOUT:   <elided>
 // CHECK:STDOUT:   %Class.elem: type = unbound_element_type %Class, %T.binding.as_type [symbolic = %Class.elem (constants.%Class.elem.bc9)]
+// CHECK:STDOUT:   %.loc15_12.2: require_specific_def_type = require_specific_def @ptr.as.Copy.impl(%T.binding.as_type) [symbolic = %.loc15_12.2 (constants.%.d8a)]
```

Comment:
The comment in `impl_lookup.cpp` says this is used when the impl is final, but I'm having trouble working out what final impl this is referring to. Am I misunderstanding something?

Reply:
It's added for all impls, used when there's a final impl involved.